### DISCUSSION TITLE
[IWP] 월드 안에서 캐릭터가 상점내 아이템을 구입하고 옷을 갈아입는 것

### DISCUSSION THREAD

**Discussion by v2form** [naverz/zepeto-studio-kor#784] [5. Q&A]
안녕하세요. 개발자 여러분 질문이 있어 여기다 글을 남기게 되었습니다. 

현재 공식 홈페이지에서 

https://studio.zepeto.me/kr/guides/making-world-products

여기까지는 마무리하여 판매가 가능하게 되었습니다. 하지만 제가 판매하고 있는 아이템들을 사용자가 맵에서 입어볼수 있고 구매까지 가능할수 있게 구현하고 싶습니다.

참고할수 있는 문서 혹은 구현할수 있는 방향을 제시해주시면 정말 도움이 될것같습니다.

잘부탁드려요

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 등록한 월드 상품을 구매 가능하게 하는 기능은 다음 가이드 문서를 참고해서 구현하실 수 있습니다.
https://studio.zepeto.me/kr/guides/in-world-purchases

월드 상품을 구매한 이후에 상품에 맞는 기능은 직접 구현하시면 되지만 (이동 속도 증가, 재화 획득, 아이템 업그레이드 등) 
제페토 캐릭터의 의상을 변경하는 기능은 아직 공식 API로 제공되지 않고 있습니다.
추후에는 제공할 예정이며 적용 시기는 미정인 점 양해 부탁 드립니다. 

감사합니다.
